srcpkgs/nestopia/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'nestopia'
 pkgname=nestopia
-version=1.50
-revision=2
+version=1.51.1
```

Comment:
Looks like 1.52 is out. Are you interested in updating this further?

Reply:
Updated and rebased on master, thanks!
Also tested on x86_64.